string    name
---
social_robot_arm_sdk/KinematicsPose pose
